modules/management/kubectl-apply/variables.tf

```diff
@@ -36,3 +36,22 @@ variable "apply_manifests" {
   }))
   default = []
 }
+
+variable "kueue" {
+  description = "Install and configure Kueue workload scheduler, more on this here https://kueue.sigs.k8s.io/docs/overview/"
+  type = object({
+    install     = optional(bool, false)
+    version     = optional(string, null)
```

Comment:
nit: I am on the fence between picking a known good version as a default or always taking the latest which I assume is what you are doing here.

Reply:
+1 on stable

Reply:
I have added the known good version in the local block where I am getting the source url for the download. I would want us not to use the LATEST version without doing some sort of integration test.

Reply:
Targeting the latest version is tricky because whenever a new version releases deployment will uninstall/install the kueue or jobset for the customers.
I believe changes in the blueprints or upgrading to a new version of gcluster (probably?) should be the expected reasons for Kueue/Jobset updates.

Reply:
It is a pinned version and as we would need to update the tolerations we can only accept the versions of kueue/jobset that we have modified, hence added a known version validation

Reply:
+1 on not picking LATEST by default.